src/Tables/TableRow.tsx

```diff
@@ -2,28 +2,48 @@ import {
   ReactElement, ReactNode,
 } from 'react';
 import styled, { withTheme } from 'styled-components';
+import { BaseTheme } from 'Theme';
 
 export interface TableRowProps {
   /** Cell components to be displayed in the row */
   children: ReactNode;
-  /** Controls whether the background of the row has a darker background color */
+  /**
+   * Controls whether the background of the row has a darker background color
+   * Inteded for use in long tables that help differentiate even/odd rows
+   * */
   isStriped?: boolean;
+  /**
+   * By default, rows will highlight on hover. Setting this to true will
+   * disable that behavior
+   */
+  noHighlight?: boolean;
```

Comment:
Yeah, I agree with your point. I decided on `noHighlight` because the default behavior was to have a highlight, and I was trying to avoid breaking too many things in the process. 

However, I will note that our eslint styles are to not pass explicit values to boolean props, so the usage would actually be:

```tsx
<TableRow noHighlight></TableRow>
```

which does avoid the double negative.

I don't feel super strongly about this, so I'd be happy to talk it over more if you still have concerns.

Reply:
No, I'm good with that. :)